pkgs/applications/search/recoll/default.nix

```diff
@@ -48,10 +48,12 @@ mkDerivation rec {
     ++ lib.optionals (!withGui) [ "--disable-qtgui" "--disable-x11mon" ]
     ++ (if stdenv.isLinux then [ "--with-inotify" ] else [ "--without-inotify" ]);
 
-  nativeBuildInputs = [ pkg-config ];
+  nativeBuildInputs = [
+    file pkg-config python3Packages.setuptools which
+  ];
 
-  buildInputs = with python3Packages; [
-    bison chmlib file python setuptools which xapian zlib
+  buildInputs = [
+    bison chmlib python3Packages.python xapian zlib
```

Comment:
```suggestion
    bison chmlib python3 xapian zlib
```

Reply:
I'd rather keep `python3Packages` so people overriding it don't have to change anything.

Reply:
Is that a common thing to do with this package? Maybe we should upstream some of those overrides if it is a very common use case.

Reply:
No idea, I've never used it.